MSBot/README.md

```diff
@@ -184,6 +184,33 @@ Here is an example invocation:
 msbot connect qna --secret EncryptItPlease --name "QnA Sauce" --kbid "cfbc14a3-9f69-4fb1-8882-e6f333691a2a" --subscriptionKey "cfbc14a3-9f69-4fb1-8882-e6f333691a2a"
 ```
 
+### Connecting to Bot Dispatch
+
+To connect your bot to bot dispatch definition you run
+
+```shell
+msbot connect dispatch [options]
+```
+
+With the following options
+
+|Option | Description|
+| ------ | ----------- |
+|-b, --bot <path>                     | path to bot file.  If omitted, local folder will look for a .bot file|
+|--secret <secret>                    | bot file secret password for encrypting service secrets|
+|-n, --name <name>                    | name for the dispatch|
```

Comment:
Do we need the name param since dispatch json already has a name?

Reply:
Never mind previous comment, this is for manual connect

Reply:
For manual dispatch connect (not from input/std in), how do we pass in the services/sources of the dispatch?